R/app.R

```diff
@@ -214,10 +214,11 @@ server <- function(input, output, session) {
                                      targetGroup = 'mappoints',
                                      circleMarkerOptions=drawCircleMarkerOptions(
                                        color="#FFFFFF",
+                                       radius=7,
                                        stroke=T,
                                        weight=2.5,
                                        fill=T,
-                                       fillColor="#ECAC7C",
+                                       fillColor="#C72424",
```

Comment:
There's a clash in greyscale with this colour and the csv point colour: https://projects.susielu.com/viz-palette?colors=[%22#C72424%22,%22#509E2F%22,%22#0078b4%22]&backgroundColor=%22white%22&fontColor=%22